JS-Closures.md

```diff
@@ -1,6 +1,6 @@
 # JavaScript Closures
 
-Closures are functions that refer to independent (free) variables (variables that are used locally, but defined in an enclosing scope). In other words, the function defined in the closure 'remembers' the environment in which it was created.
+Closures are functions that refer to independent (free) variables. Free variables are variables that are used locally, but defined in an enclosing scope. In other words, the function defined in the closure 'remembers' the environment in which it was created.
```

Comment:
The 'closure' in the last sentence is different to the 'closure' in the first sentence.